test/accesskey.test.js

```diff
@@ -0,0 +1,228 @@
+/*
```

Comment:
Nit, non-blocker: Maybe this should be renamed to: `temporary-accesskey.test.js`? Or combined with `accesskeys.test.js`?